src/Toolkit/Toolkit/UI/Controls/FeatureForm/TextFormInputView.cs

```diff
@@ -41,14 +56,20 @@ public TextFormInputView()
 
         private void ConfigureTextBox()
         {
+#if WINDOWS_XAML
+            VisualStateManager.GoToState(this, Element?.IsEditable == false ? "Disabled" : "Enabled", true);
+            VisualStateManager.GoToState(this, Element?.Input is BarcodeScannerFormInput && Element.IsEditable ? "ShowBarcode" : "HideBarcode", true);
+            VisualStateManager.GoToState(this, Element?.IsEditable == true && Element?.Input is TextAreaFormInput ? "MultiLineText" : "SingleLineText", true);
```

Comment:
What about non-editable "TextArea" inputs?  Shouldn't they be multi-line as well?  For example, in the sample, try writing a long description and see how "Place Summary" (read-only TextArea) is getting cut off -- no wrapping and no scrolling:

![2024-08-02_000456 Toolkit SampleApp WinUI](https://github.com/user-attachments/assets/cefccf69-844d-400f-9592-327a3b064284)

This is different from AGOL, which has both wrapping and scrollbars:

![2024-08-02_000728 msedge](https://github.com/user-attachments/assets/0d7ef2c0-508f-49e4-8c6e-20584e4e1f65)